assets/app/styles/_create.less

```diff
@@ -0,0 +1,225 @@
+//
+// Create App
```

Comment:
Please be careful about keeping any updates to `core.less` when you rebase since you've moved parts of it to other files.